fontbe/src/error.rs

```diff
@@ -95,6 +95,8 @@ pub enum Error {
     CmapConflict(#[from] CmapConflict),
     #[error("Progress stalled computing composite bbox: {0:?}")]
     CompositesStalled(Vec<GlyphName>),
+    #[error("Inconsistent palette lengths observed: {0:?}")]
```

Comment:
I think it's useful to note that this is a CPAL thing, so that the debugger knows where to start looking

Reply:
(removed due to invariant movements)